README.md

```diff
@@ -48,3 +36,10 @@ To use the sysroot, use the `--sysroot=` option:
 ```
 
 to run the compiler using the newly built sysroot.
+
+Note that Clang packages typically don't include cross-compiled builds of
+compiler-rt, `libclang_rt.builtins-wasm32.a`, so they may not be usable without
```

Comment:
How about generalizing: `Clang packages typically don't include cross-compiled builds of standard libraries such as compiler-rt or libc++abi` since its not just compiler-it

Reply:
Done!